src/Explorer/Tabs/SettingsTab.ts

```diff
@@ -1617,17 +1564,15 @@ export default class SettingsTab extends TabsBase implements ViewModels.WaitsFor
       this.GEOMETRY;
     this.geospatialConfigType.setBaseline(geospatialConfigType);
 
-    if (!this.hasAutoPilotV2FeatureFlag()) {
-      const maxThroughput =
-        this.collection &&
-        this.collection.offer &&
-        this.collection.offer() &&
-        this.collection.offer().content &&
-        this.collection.offer().content.offerAutopilotSettings &&
-        this.collection.offer().content.offerAutopilotSettings.maxThroughput;
+    const maxThroughput =
+      this.collection &&
+      this.collection.offer &&
+      this.collection.offer() &&
+      this.collection.offer().content &&
+      this.collection.offer().content.offerAutopilotSettings &&
+      this.collection.offer().content.offerAutopilotSettings.maxThroughput;
 
-      this.autoPilotThroughput(maxThroughput || AutoPilotUtils.minAutoPilotThroughput);
-    }
+    this.autoPilotThroughput(maxThroughput || AutoPilotUtils.minAutoPilotThroughput);
```

Comment:
nit: if `maxThroughput` is equal to zero, it will switch to use the constant. But probably an unlikely case.

Reply:
Agreed. Unlikely.